api/envoy/config/trace/v3/opentelemetry.proto

```diff
@@ -29,6 +29,12 @@ message OpenTelemetryConfig {
   // The upstream HTTP cluster that will receive OTLP traces.
   // This field can be left empty to disable reporting traces to the HTTP service.
   // Only one of ``grpc_service``, ``http_service`` may be used.
+  //
+  // Note: The ``request_headers_to_add`` property in the OTLP HTTP exporter service
+  // does not support the :ref:`format specifier <config_access_log_format>` as used for
+  // :ref:`HTTP access logging <config_access_log>`.
+  // The values configured are added as HTTP headers on the OTLP export request
+  // without any formatting applied.
```

Comment:
Please use `.. note::` block. You can find lots of example from exist docs

Reply:
Done in [cb08695](https://github.com/envoyproxy/envoy/pull/29207/commits/cb0869505958ef8c54c174ea849e989716535720)

![image](https://github.com/envoyproxy/envoy/assets/5938087/6b7307ff-e5d2-476a-ab18-35bd6aafc88d)